p/themes/Dark-pink/pinkdark.css

```diff
@@ -0,0 +1,125 @@
+.btn:hover {
+	background: unset !important;
```

Comment:
Is is possible to reduce the use of the many `!important` or are they actually needed?

Reply:
I think they are actually needed, I can test without them tho

Reply:
Yep, `!important` were unnecessary, removed them